src/index.js

```diff
@@ -16,13 +16,15 @@ import {
 } from 'react-router-dom';
 
 import { Security } from '@okta/okta-react';
+import { Auth0Provider } from '@auth0/auth0-react'; //this will replace okta above
```

Comment:
You say 'this **will** replace Okta above, but I see it implemented further down in the code. Is there reasoning for keeping the old systems in place? From what I can tell you've swapped all of the secureRoutes to protectedRoutes in this file.

Reply:
Absolutely! I will remove this line and the wrapper around the components as it has been replaced with Auth0 ProtectedRoute